### ISSUE TITLE
Can't find locales for I18n

### ISSUE THREAD

**Issue by jordigg** [zpaulovics/datetimepicker-rails#34]
Hi all,

I would like to translate datetimepicker-rails I follow the readme but the locales aren't included. I've tried with the originals from plugin but doesn't work. Does anyone know how to make it work?

I can change few parameters on the YML file but I want to translate the ui of the plugin.

Thanks!

**Comment by zpaulovics** [comment]
Hi,

You can find the locales in the MomentJs package used by datetime picker.


Üdv,

Zoli

**Comment by jordigg** [comment]
Thanks for the answer. I removed the gem because I had momentjs installed but without locales. I tried installing datetimepicker-rails again after removing it completely and by adding the gem 'momentjs-rails', '>= 2.8.1', :github => 'derekprior/momentjs-rails' manually. I t didn't work, moment.js gets installed but the locales don't. I can find them on the assets folder.

Using the default moment.js path for the locales works fine: //= require moment/<locale>.js

Best,
Jordi
